utils/badPractices.ts

```
@@ -0,0 +1,114 @@
+// Utility file with bad practices for CodeRabbit to catch
+
+// Global variable - bad practice
+var globalCounter = 0
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'let' instead.


(lint/style/noVar)